serial-vault-services-snap/install.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
```

Comment:
It would be nice having some check to see if current user is root and exit if not. 

if [ "$EUID" -ne 0 ]
  then echo "Please run as root"
  exit
fi

or 

if [ $(id -u) -ne 0 ]
  then echo "Please run as root"
  exit
fi